### PR TITLE
Update Readme

### PR DESCRIPTION
**Pull request by ralphtheninja**
Closes https://github.com/Level/level-browserify/issues/36

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -36,44 +37,335 @@ db.put('name', 'Level', function (err) {
...
 <tr><th align="left">Pedro Teixeira</th><td><a href="https://github.com/pgte">GitHub/pgte</a></td><td><a href="https://twitter.com/pgte">Twitter/@pgte</a></td></tr>
 <tr><th align="left">James Halliday</th><td><a href="https://github.com/substack">GitHub/substack</a></td><td><a href="https://twitter.com/substack">Twitter/@substack</a></td></tr>
+<tr><th align="left">Vincent Weevers</th><td><a href="https://github.com/vweevers">GitHub/vweevers</a></td><td><a href="https://twitter.com/vweevers">Twitter/@vweevers</a></td></tr>
 </tbody></table>
 
 
-<a name="licence"></a>
-Licence &amp; copyright
--------------------
+## Licence
```
Comment (vweevers) [inline README.md]: c/s

Comment (ralphtheninja) [reply README.md]: Good catch!

**Comment by vweevers** [comment]
Perhaps a TOC?

**Comment by ralphtheninja** [comment]
@vweevers Aye, I'll add an issue.

**Pull request merged**